packages/backend-core/src/redis/utils.ts

```diff
@@ -27,6 +27,7 @@ export enum Databases {
   GENERIC_CACHE = "data_cache",
   WRITE_THROUGH = "writeThrough",
   LOCKS = "locks",
+  SOCKET_IO = "socket_io",
```

Comment:
This redis namespace is never used because socket.io generates it's own key names, so I've literally only added this to be consistent with other usages of constructing instances of our redis wrapper, as technically we could still manually use that client instance to send changes. So I think it's still prudent to reserve a dedicated namespace.

Reply:
With this - is there anyway we could get the socket_io layer to use its own Redis database? This way its in a completely separate keyspace and won't show up in any key/scan lists, could help with performance of certain features which enumeration a lot of keys.

Reply:
Since it's only using redis for pub/sub, I'm not sure anything it actually sets is persistent. It's all just ephemeral messages which are sent to subscribers but then dropped. At least I'm pretty sure that's how it works.

Reply:
ah I see - so it doesn't show up in the keyspace at all, thats perfect then it won't affect anything else in terms of performance!